k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -841,42 +838,48 @@ function deployCortxData()
-            --set cortxdata.name="cortx-data-${node_name}" \
-            --set cortxdata.image="${cortxdata_image}" \
-            --set cortxdata.nodeselector="${node_selector}" \
-            --set cortxdata.service.clusterip.name="cortx-data-clusterip-svc-${node_name}" \
-            --set cortxdata.service.headless.name="cortx-data-headless-svc-${node_name}" \
-            --set cortxdata.cfgmap.volmountname="config001-${node_name}" \
-            --set cortxdata.machineid.value="${cortxdata_machineid}" \
-            --set cortxdata.localpathpvc.name="cortx-data-fs-local-pvc-${node_name}" \
-            --set cortxdata.localpathpvc.mountpath="${local_storage}" \
-            --set cortxdata.motr.numiosinst=${#cvg_index_list[@]} \
-            --set cortxdata.motr.startportnum="$(extractBlock 'solution.common.motr.start_port_num')" \
-            --set cortxdata.hax.port="$(extractBlock 'solution.common.hax.port_num')" \
-            --set cortxdata.motr.resources.requests.memory="$(extractBlock 'solution.common.resource_allocation.data.motr.resources.requests.memory')" \
-            --set cortxdata.motr.resources.requests.cpu="$(extractBlock 'solution.common.resource_allocation.data.motr.resources.requests.cpu')" \
-            --set cortxdata.motr.resources.limits.memory="$(extractBlock 'solution.common.resource_allocation.data.motr.resources.limits.memory')" \
-            --set cortxdata.motr.resources.limits.cpu="$(extractBlock 'solution.common.resource_allocation.data.motr.resources.limits.cpu')" \
-            --set cortxdata.confd.resources.requests.memory="$(extractBlock 'solution.common.resource_allocation.data.confd.resources.requests.memory')" \
-            --set cortxdata.confd.resources.requests.cpu="$(extractBlock 'solution.common.resource_allocation.data.confd.resources.requests.cpu')" \
-            --set cortxdata.confd.resources.limits.memory="$(extractBlock 'solution.common.resource_allocation.data.confd.resources.limits.memory')" \
-            --set cortxdata.confd.resources.limits.cpu="$(extractBlock 'solution.common.resource_allocation.data.confd.resources.limits.cpu')" \
-            --set cortxdata.hax.resources.requests.memory="$(extractBlock 'solution.common.resource_allocation.hare.hax.resources.requests.memory')" \
-            --set cortxdata.hax.resources.requests.cpu="$(extractBlock 'solution.common.resource_allocation.hare.hax.resources.requests.cpu')" \
-            --set cortxdata.hax.resources.limits.memory="$(extractBlock 'solution.common.resource_allocation.hare.hax.resources.limits.memory')" \
-            --set cortxdata.hax.resources.limits.cpu="$(extractBlock 'solution.common.resource_allocation.hare.hax.resources.limits.cpu')" \
-            -n "${namespace}" \
-            || exit $?
-    done
+    local -r cortx_data_values_file=cortx-data-values.yaml
+
+    yq --null-input "
+        .cortxdata.replicas=\"${data_node_count}\"
```

Comment:
```suggestion
        .cortxdata.replicas = ${data_node_count}
```
Quoting it results in `replicas: "3"` instead of `replicas: 3`. The latter is the same as the values file definition.